provisioner/powershell/provisioner.go

```diff
@@ -112,7 +115,12 @@ func (p *Provisioner) defaultExecuteCommand() string {
 		return baseCmd
 	}
 
-	return fmt.Sprintf(`powershell -executionpolicy %s "%s"`, p.config.ExecutionPolicy, baseCmd)
+	if(p.config.Pwsh){
+	  return fmt.Sprintf(`pwsh -executionpolicy %s -command "%s"`, p.config.ExecutionPolicy, baseCmd)
+	} else {
+	  return fmt.Sprintf(`powershell -executionpolicy %s "%s"`, p.config.ExecutionPolicy, baseCmd)
```

Comment:
Re-reading through this PR, I notice the indentation is off here, could you replace the two-spaces by a tab? That'd help it stay consistent with the rest of the code and the Go guidelines.